exercises/practice/phone-number/.meta/example.cairo

```diff
@@ -4,58 +4,55 @@ pub fn clean(phrase: ByteArray) -> ByteArray {
     let mut i = 0;
     while i < phrase.len() {
         let c = phrase[i];
-        i += 1;
-        if c >= '0' && c <= '9' {
+
+        if test_valid(c) && is_numeric(c) {
```

Comment:
Since `test_valid` can either return `true` or panic, you can put it above the the `if` statement, making it a type of an assert wrapper.

```rust
assert_valid(c); // if ok, does nothing; otherwise panic
if is_numeric(c) {
...
```